test/inductor/test_torchinductor.py

```diff
@@ -5360,7 +5360,6 @@ def fn(a, b):
         )
         self.assertEqual(torch._inductor.metrics.generated_kernel_count, 0)
 
-    @config.patch(search_autotune_cache=False)
```

Comment:
is this intentional?

Reply:
Sorry, I mistook this test for dup-args case.